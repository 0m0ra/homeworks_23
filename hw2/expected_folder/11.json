{"error": "Input file does not exist."}