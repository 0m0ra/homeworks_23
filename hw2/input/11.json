{
    "user": {
        "region": "Moscow",
        "registered": "201117",
        "last_login": "2023-12-11",
        "email": "user@yandex.ru",
        "age": 71
    },
    "user2": {
        "region": "Vladivostok",
        "registered": "2009-11-25",
        "last_login": "2022-12-15",
        "email": "user2@gmail.com",
        "age": 56
    }
}