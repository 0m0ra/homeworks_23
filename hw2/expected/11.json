"The registration number entered is of the wrong type"